INSTALL.md

```diff
@@ -69,9 +69,11 @@ Additionaly, Visual Studio is required to compile in Windows.
 
         *Visual Studio* is the only compiler officially supported to build *Kratos* under *Windows*. The minimium required version is Visual Studio 2017, but we recommend to use Visual Studio 2019 or higher. 
 
-        * [Download Visual Studio](https://visualstudio.microsoft.com/en/thank-you-downloading-visual-studio/?sku=Community&rel=15)
+        * [Download Visual Studio](https://visualstudio.microsoft.com/en/thank-you-downloading-visual-studio/?sku=Community&rel=16)
 
-        Since *Visual Studio* is a multi-language IDE, some distributions come without C++ compiler. Please, make sure that you can create a C++ project before continuing, in case C++ packages were missing you will be prompt to download them.
+        Since *Visual Studio* is a multi-language IDE, some distributions come without C++ compiler. Please, make sure that you can create a C++ project before continuing, in case C++ packages were missing you will be prompt to download them. You can install the <span style="color:red">Desktop development with C++</span> workload with the Visual Studio Installer to acquire all necessary depencencies to compile C++ projects. 
```

Comment:
`<span style="color:red">` seems not to work

Reply:
True! I have replaced it with bold text